### PR TITLE
Added NewsApiClient

### PR DESCRIPTION
**Pull request by Bekkie**
I could not resist adding your NewsApiClient support to the printer monitor myself.

### CODE REVIEW

**Comment by Qrome** [comment]
Oh wow, you have done it now.  :)  I will have to take a close look at this over the next week.

File: printermonitor/printermonitor.ino [hunk 1]
```diff
@@ -66,7 +66,7 @@ void drawClockHeaderOverlay(OLEDDisplay *display, OLEDDisplayUiState* state);
 // Set the number of Frames supported
 const int numberOfFrames = 3;
 FrameCallback frames[numberOfFrames];
-FrameCallback clockFrame[2];
+FrameCallback clockFrame[20];
```
Comment (derpicknicker1) [inline printermonitor/printermonitor.ino]: Why do you reserve 20 when only 13 are needed?

File: printermonitor/printermonitor.ino [hunk 2]
```diff
@@ -90,6 +90,32 @@ int printerCount = 0;
 // Weather Client
 OpenWeatherMapClient weatherClient(WeatherApiKey, CityIDs, 1, IS_METRIC);
 
+// News Client
+NewsApiClient newsClient(NEWS_API_KEY, NEWS_SOURCE);
+int newsIndex = 0;
+
+String NEWS_OPTIONS = "<option>bbc-news</option>"
```
[the remark this answers is not in the document]
Comment (Bekkie) [reply printermonitor/printermonitor.ino]: I (partly) agreed with your string idea, but i took this list from your marquee project and added a news source to this list.

File: printermonitor/printermonitor.ino [hunk 3]
```diff
@@ -805,10 +927,90 @@ void drawWeather(OLEDDisplay *display, OLEDDisplayUiState* state, int16_t x, int
 
   display->setFont(ArialMT_Plain_16);
   display->drawString(0 + x, 24 + y, weatherClient.getCondition(0));
-  display->setFont((const uint8_t*)Meteocons_Plain_42);
+  display->setFont(Meteocons_Plain_42);
```
[the remark this answers is not in the document]
Comment (Bekkie) [reply printermonitor/printermonitor.ino]: I will put it back in, but then I will not compile on my configuration
"invalid conversion from 'const uint8_t* {aka const unsigned char*}' to 'const char*' [-fpermissive]"

**Comment by Qrome** [comment]
Thanks for this submission, however in keeping with my goals to try and keep this project focused on the 3D printing monitor, I will be closing this PR.   Thanks!

**Pull request closed without merging**